markdown/developers/tutorials/adding_modules_to_pipelines.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Tutorial: Adding Modules to Pipelines'
+subtitle: Quick reference of the steps needed to add a new module to a pipeline.
+---
+
+This tutorial is a quick reference overview of the steps you will normally take when adding a new module to a pipeline.
+
+Different pipelines may have different workflows, but the following steps will cover the most common aspects.
+
+1. Install module to the pipeline
+
+   ```bash
+   nf-core modules install <tool>/<subtool>
+   ```
+
+2. Add `include` statement at the top of the pipeline's (sub)workflow to import module
```

Comment:
Not sure if the include part would work as is in a subworkflow given the relative path

Reply:
But you always have to have an include statement no?

https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/bam_markduplicates_picard/main.nf

Reply:
Could mention that a suggestion for this comes from the install command.